modules/shibboleth/templates/shibboleth2.xml.erb

```diff
@@ -65,6 +65,14 @@
                     <md:GivenName>Unity SSO Support</md:GivenName>
                     <md:EmailAddress>unity-sso@researchresearch.com</md:EmailAddress>
                 </md:ContactPerson>
+
+                <mdui:UIInfo>
+                    <mdui:DisplayName xml:lang="en">*Unity</mdui:DisplayName>
+                    <mdui:Description xml:lang="en">*Unity is the new way to work with anyone in universities and research. It’s a simple place to form groups, share files and work on them together.</mdui:Description>
+                    <mdui:InformationURL xml:lang="en">http://www.unity.ac</mdui:InformationURL>
+                    <mdui:Logo height="60" width="80">https://shib-sp.unity.ac/assets/unity-sso.png</mdui:Logo>
+                </mdui:UIInfo>
```

Comment:
Just realized that the mdui xmlns probably needs to be imported where the `md` namespace is. See the `xmlns:md="..."` attribute on the root tag.